after/syntax/typescript/graphql.vim

```diff
@@ -42,6 +42,15 @@ exec 'syntax match graphqlTaggedTemplate +' . s:tags . '\ze`+ nextgroup=graphqlT
 " Support expression interpolation ((${...})) inside template strings.
 syntax region graphqlTemplateExpression start=+${+ end=+}+ contained contains=typescriptTemplateSubstitution containedin=graphqlFold keepend
 
+" support #graphql , #gql comment strings
+syntax region graphqlTemplateString
+      \ start=+`\(#\s\{,4\}\(gql\|graphql\)\)\@=+
+      \ skip=+\\\\\|\\`+
+      \ end=+`+me=s-1
```

Comment:
I've had trouble with adjusting the match range (e.g. `me`) before, but if this is what it takes to get things working, I'm good with it.

Reply:
A spent a decent chunk of time reading through :help syntax and it's still some what mystical to me. I understand why this approach works but for the life of me I couldn't grok why the approach used in the JS syntax files didn't work here.

Perhaps with time some other contributor will find a more appropriate approach.